src/backend/InvenTree/users/test_tokens.py

```
@@ -0,0 +1,68 @@
+        self.assertEqual(authenticated_token.version, 2)
+
+    def test_v2_token_with_peppers(self):
+        with self.settings(API_TOKEN_PEPPERS={'1': 'pepper1', '2': 'pepper2'}):
+            key = ApiToken.generate_key()
+            token = ApiToken.objects.create(user=self.user, key=key, version=2)
+            self.assertEqual(token.pepper_id, '2')
+            user, authenticated_token = self.auth.authenticate_credentials(key)
+            self.assertEqual(user, self.user)
+
+            with self.settings(API_TOKEN_PEPPERS={'1': 'pepper1', '2': 'pepper2', '3': 'pepper3'}):
+                user, authenticated_token = self.auth.authenticate_credentials(key)
```

Ruff (0.15.15)

[warning] Unpacked variable `authenticated_token` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)